docs/observability.md

```diff
@@ -0,0 +1,126 @@
+# COS Integration
+
+## (User) Getting Started
+
+This section describes how to deploy a single-node MicroK8s cluster with COS, and then relate MicroK8s with COS to get metrics, logs and dashboards for the cluster.
+
+### Deploy MicroK8s
+
+```bash
+# Add a machine model and deploy MicroK8s
+juju add-model microk8s
+juju deploy microk8s --channel edge --config hostpath_storage=true --constraints 'mem=8G'
+juju expose microk8s
+
+# Add MicroK8s as Kubernetes cloud to Juju
+juju run --unit microk8s/leader -- microk8s config | juju add-k8s k8s --controller "$(juju controller-config controller-name)"
```

Comment:
You may want to get the IP of the API endpoint from juju here. This is needed for example on AWS clouds.

Reply:
This kubeconfig will contain the internal IP of the microk8s node, which _should_ be reachable from the Juju controller instance in all cases.